plugins/csp-lod-bodies/src/LodBody.cpp

```diff
@@ -23,19 +23,22 @@ namespace csp::lodbodies {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-LodBody::LodBody(std::shared_ptr<cs::core::Settings> const& settings,
-    std::shared_ptr<cs::core::GraphicsEngine>               graphicsEngine,
-    std::shared_ptr<cs::core::SolarSystem>                  solarSystem,
-    std::shared_ptr<Plugin::Settings> const&                pluginSettings,
-    std::shared_ptr<cs::core::GuiManager> const&            pGuiManager,
-    std::shared_ptr<GLResources> const& glResources, std::string const& anchorName)
+LodBody::LodBody(
+    std::shared_ptr<cs::core::Settings> const&      settings,
+    std::shared_ptr<cs::core::GraphicsEngine>       graphicsEngine,
+    std::shared_ptr<cs::core::SolarSystem>          solarSystem,
+    std::shared_ptr<Plugin::Settings> const&        pluginSettings,
+    std::shared_ptr<cs::core::GuiManager> const&    pGuiManager,
+    std::shared_ptr<GLResources> const&             glResources,
+    std::string const&                              anchorName
+    )
```

Comment:
This looks like you formatted this manually? Maybe you should run the `tools/clang-format.sh` script to fix all formatting issues.

And then make a separate commit with the message `:sparkles: Apply clang-format`.